ragna/source_storages/_embedding_model.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC, abstractmethod
+
+from sentence_transformers import SentenceTransformer
+import torch
```

Comment:
PyTorch is a massive dependency that we cannot pull in by default. This has to be optional.